extensions/pdh/PerformanceDataMonitor.cpp

```diff
@@ -45,9 +45,15 @@ core::Property PerformanceDataMonitor::CustomPDHCounters(
 core::Property PerformanceDataMonitor::OutputFormatProperty(
     core::PropertyBuilder::createProperty("Output Format")->
     withDescription("Format of the created flowfiles")->
-    withAllowableValue<std::string>(JSON_FORMAT_STR)->
-    withAllowableValue(OPEN_TELEMETRY_FORMAT_STR)->
-    withDefaultValue(JSON_FORMAT_STR)->build());
+    withAllowableValue(PRETTY_JSON_FORMAT_STR)->
+    withAllowableValue(COMPACT_JSON_FORMAT_STR)->
+    withAllowableValue(PRETTY_OPEN_TELEMETRY_FORMAT_STR)->
+    withAllowableValue(COMPACT_OPEN_TELEMETRY_FORMAT_STR)->
+    withDefaultValue(PRETTY_JSON_FORMAT_STR)->build());
+
+core::Property PerformanceDataMonitor::DoublePrecisionProperty(
+  core::PropertyBuilder::createProperty("Double Precision")->
+  withDescription("Rounds the double values to this precision (blank for maximum precision)")->build());
```

Comment:
What does precision mean in this context? Some power of 10 decimal place? Or power of 2? Or number of significant decimal digits? Or ...
This could use some more precise description.

Reply:
Good point, changed it to "The number of decimal places to round the values to (blank for no rounding)" in [26802f7](https://github.com/martinzink/nifi-minifi-cpp/commit/26802f77ef539757fcedbffea0f596b2ef0cc828)